core/src/main/java/com/wttech/aem/acm/core/instance/InstanceStateService.java

```diff
@@ -0,0 +1,43 @@
+package com.wttech.aem.acm.core.instance;
+
+import com.wttech.aem.acm.core.code.ExecutionQueue;
+import com.wttech.aem.acm.core.osgi.OsgiContext;
+import java.util.TimeZone;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Reference;
+
+@Component(immediate = true, service = InstanceStateService.class)
```

Comment:
Instance-related things, InstanceInfo ... was designed to handle them all; do not create second thing for the same thing